test/e2e/e2e_suite_test.go

```diff
@@ -83,6 +83,8 @@ var _ = BeforeSuite(func() {
 	k8sClient, err = ctrl.New(config, ctrl.Options{Scheme: scheme.Scheme})
 	Expect(err).NotTo(HaveOccurred())
 
+	os.Setenv("DEPLOYMENT_NAMESPACE", operatorInstalledNamespcae)
```

Comment:
Why is this needed in the e2e test? 🤔

Reply:
Function _GetFenceAgentsRemediationPod_ calls function _GetDeploymentNamespace_  which is looking for the namespace value  from **DEPLOYMENT_NAMESPACE** envioronment variable. 
Without this line I receive an error (see below) and the e2e test fails with [timeout](https://github.com/medik8s/fence-agents-remediation/blob/main/test/e2e/far_e2e_test.go#L239-L246).

Error:

> failed to get pod. Might try again	{"error": "failed fetching FAR namespace - DEPLOYMENT_NAMESPACE must be set"}

Reply:
nevermind, found it, it's indirectly needed by the test code as well, and not just controller code

Reply:
So lgtm ? :)